doc/source/structures/misc/string.rst

```diff
@@ -127,6 +169,9 @@ Structure
         * - :meth:`INSERT(index, string)`
           - :struct:`String`
           - Returns a new string with the given string inserted at the given index into this string
+        * - :attr:`ITERATOR`
```

Comment:
Oh hey, you're right.  It is literally just an iterator suffix, not the rest of the enumerator stuff... I take it all back.  I'll just remove the enumerator portion of that attribute definition below instead.  Ignore my craziness.